BURNER/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -e
+
+$LINKER_PATH/tvm_linker compile BURNER.code --abi-json BURNER.abi.json --data data.boc -o BURNER.tvc
```

Comment:
how did you build the file `data.boc`? 
you need to add an explanation to the document or script

Reply:
+

Reply:
It's an empty cell serialized to BoC.